### PR TITLE
javascript only implementation

### PR DESCRIPTION
**Pull request by yigalirani**
hi, 

This is my first ever pull request on github:)

I have wrote a new implementation for you wonderful todomvc . It uses only javascript with no external dependencies. I hope you like it

-yigal:)

### CODE REVIEW

**Comment by anubhav7495** [comment]
http://todomvc.com/examples/vanillajs
Looks like Pure Javascript to me.

**Comment by yigalirani** [comment]
@anubhav7495: True, both VanillaJS and SimpleJS are pure JavaScript implementations.

However, SimpleJS has something that VanillaJS lacks: it truly is a non-MVC solution. Unlike VanillaJS, it is not structured using Models, Controllers, Views and routers. Instead, it is implemented in the simplest way possible and can be useful for anyone who wants to establish their JavaScript skills before learning MVC and MVC frameworks.

As an added bonus, it is much more concise: It has only one .js file with 224 sloc where VanillaJS has 7 files and 805 sloc

**Comment by samccone** [comment]
@yigalirani cool implementation!, I would love for you to roll this work back into the initial js only implementation, perhaps 1 simplification at a time, as compared to having another only JS implementation.


thoughts?

**Comment by yigalirani** [comment]
@samccone, Thanks, I am glad you like it. 
I wouldn’t want to change anything with VanillaJS as it is just fine for what it is: an implementation without dependencies that does use the MVC pattern. 
The two implementations are also too different to be merged. My suggestion is to add mine as yet another dependencies-free implementation.

File: examples/simplejs/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+			</section>
+			<footer id="footer"></footer>
+		</section>
+		<footer id="info">
+			<p>Double-click to edit a todo</p>
+			<p>Created by <a href="http://symbolclick.com">Yigal Irani</a></p>
+			<p>Part of <a href="http://todomvc.com">TodoMVC</a></p>
+		</footer>
+		<script id="todo-template" type="text/template">
+			<li #liClass data-id="#id" id="#id">
```
Comment (dmitriz) [inline examples/simplejs/index.html]: Is `#liClass` a valid attribute? Is it backward compatible? Is it needed vs only standard attr names?

Comment (dmitriz) [inline examples/simplejs/index.html]: Sorry, now realized it is a template

**Comment by dmitriz** [comment]
Have you run the tests?

    concurrent "gulp test-server" "npm run test -- --framework=simplejs"

 I am getting many failures:

```
[1]   TodoMVC - simplejs
[1]     When page is initially opened
[1] 
[1]       ✓ should focus on the todo input field
[1] [ Object {
[1]     level: Object { name: 'SEVERE', value: 1000 },
[1]     message: 'http://localhost:8000/examples/simplejs/node_modules/todomvc-common/base.css 0:0 Failed to load resource: the server responded with a status of 404 (Not Found)',
[1]     timestamp: 1450925455577,
[1]     type: '',
[1]     closure_uid_552538734: undefined },
[1]   Object {
[1]     level: Object { name: 'SEVERE', value: 1000 },
[1]     message: 'http://localhost:8000/examples/simplejs/node_modules/todomvc-app-css/index.css 0:0 Failed to load resource: the server responded with a status of 404 (Not Found)',
[1]     timestamp: 1450925455577,
[1]     type: '',
[1]     closure_uid_552538734: undefined } ]
[1]     No Todos
[1] 
[1]       ✓ should hide #main and #footer (121ms)
[1] [ Object {
[1]     level: Object { name: 'SEVERE', value: 1000 },
[1]     message: 'http://localhost:8000/examples/simplejs/node_modules/todomvc-common/base.css 0:0 Failed to load resource: the server responded with a status of 404 (Not Found)',
[1]     timestamp: 1450925457429,
[1]     type: '',
[1]     closure_uid_552538734: undefined },
[1]   Object {
[1]     level: Object { name: 'SEVERE', value: 1000 },
[1]     message: 'http://localhost:8000/examples/simplejs/node_modules/todomvc-app-css/index.css 0:0 Failed to load resource: the server responded with a status of 404 (Not Found)',
[1]     timestamp: 1450925457429,
[1]     type: '',
[1]     closure_uid_552538734: undefined } ]
[1]     New Todo
[1] 
[1]       ✓ should allow me to add todo items (1107ms)
[1] [ Object {
[1]     level: Object { name: 'SEVERE', value: 1000 },
[1]     message: 'http://localhost:8000/examples/simplejs/node_modules/todomvc-common/base.css 0:0 Failed to load resource: the server responded with a status of 404 (Not Found)',
[1]     timestamp: 1450925459549,
[1]     type: '',
[1]     closure_uid_552538734: undefined },
[1]   Object {
[1]     level: Object { name: 'SEVERE', value: 1000 },
[1]     message: 'http://localhost:8000/examples/simplejs/node_modules/todomvc-app-css/index.css 0:0 Failed to load resource: the server responded with a status of 404 (Not Found)',
[1]     timestamp: 1450925459549,
[1]     type: '',
[1]     closure_uid_552538734: undefined } ]
 ....
```

**Comment by yigalirani** [comment]
@dmitriz, Yes i did. 

Also, it is actually impossible to submit a pull request without passing the tests as the server runs the tests (and style checks) automatically.

About the reported problem, it seems like there is a missing css file from your installation. To fetch it,  run "npm install" at your examples/simplejs  directory.

**Comment by dmitriz** [comment]
I was thinking Travis only runs the `jscs` tests but yes - you are right, my bad.

**Comment by addyosmani** [comment]
Thank you for your PR. Master currently has both a recent vanilla JS implementation https://github.com/tastejs/todomvc/tree/master/examples/vanillajs and an ES2015 one that we're happy with - this covers both the ES5 and JS.next bases.

My concern with landing this new work is it opens up the gates to N different permutations on vanilla being submitted in by other members of the community. I don't feel that helps folks comparing approaches as the "should I just be exploring vanilla?" question is already provided for with what we have today. 

With that in mind, I've decided to close this PR. I'd like TodoMVC to focus on landing new implementations that either update existing apps or are very different from what we have today (more functional programming takes in JS would be interesting).

**Pull request closed without merging**